README.md

```diff
@@ -105,6 +105,8 @@ After you're finished please remove all the comments and instructions!
 - **[The data is preserved in the browser's memory by using localStorage]**
 - **[Use JavaScript classes instead of objects and arrays]**
 - **[Create class methods to add, remove and edit tasks]**
+- **[Clear all tasks that are marked done by a single click]**
+- **[Strikes through completed tasks]**
```

Comment:
- [ ] it would be better to add them under the `future features` section so that don't let any section empty and also it's their place